runtime/moonbeam/src/lib.rs

```diff
@@ -1311,6 +1311,20 @@ impl pallet_multisig::Config for Runtime {
 	type WeightInfo = moonbeam_weights::pallet_multisig::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const MaxStorageRoots: u32 = 10;
```

Comment:
It might be too short in practice, especially with async backing, when the block production is perfect 0 roots will mean 1 min, might be very short for dapps and user signing validation, can we increase this constant to 30, or directly move it to storage with the parameter pallet?

Reply:
done